### PR TITLE
Draft: ICompiler and IExecutors (WIP)

### PR DESCRIPTION
**Pull request by hseok-oh**
Draft for `ICompiler` and `IExecutors`

Related issue: #10000

### CODE REVIEW

**Comment by hseok-oh** [comment]
@nnfw-bot test android-runtime-builder

File: runtime/onert/core/src/exec/SingleModelExecutors.h [hunk 1]
```diff
@@ -0,0 +1,63 @@
+
```
Comment (ragmani) [inline runtime/onert/core/src/exec/SingleModelExecutors.h:1]:
```suggestion
```

File: runtime/onert/core/include/compiler/Compiler.h [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (ragmani) [inline runtime/onert/core/include/compiler/Compiler.h:2]:
```suggestion
 * Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved
```

File: runtime/onert/core/src/exec/MultiModelExecutors.h [hunk 3]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (ragmani) [inline runtime/onert/core/src/exec/MultiModelExecutors.h:1]:
```suggestion
```

**Pull request closed without merging**